cloudinit/util.py

```diff
@@ -68,17 +68,28 @@
                    ['running-in-container'],
                    ['lxc-is-container'])
 
-PROC_CMDLINE = None
+_CACHED_RESPONSES = {}
```

Comment:
I don't think we need this definition any more.

Reply:
i was so sure i had removed that…